includes/class-ssp-frontend.php

```diff
@@ -132,11 +146,11 @@ public function ssp_override_player_styles(){
 	 * Enqueue styles and scripts
 	 */
 	public function load_styles_and_scripts(){
-		wp_enqueue_style( 'google-font-robotto' , '//fonts.googleapis.com/css?family=Roboto:400,700', array(), SSP_VERSION);
```

Comment:
Commenting out but leaving the older script loading methods in case we return to them.